delorean/interface.py

```diff
@@ -10,23 +10,47 @@
 UTC = "UTC"
 utc = timezone("utc")
 
+NAIVEMODE_UTC = "utc"
+NAIVEMODE_NAIVE = "naive"
+NAIVEMODE_RAISE = "raise"
+NAIVE_MODES = (NAIVEMODE_UTC, NAIVEMODE_NAIVE, NAIVEMODE_RAISE)
 
-def parse(s, dayfirst=True, yearfirst=True):
+def parse(s, dayfirst=True, yearfirst=True, naivemode=NAIVEMODE_UTC):
```

Comment:
It only pertains to naive inputs, though. Fi you add some other conditional behavior later, what would you do with naming?